platforms/hyperledger-indy/configuration/samples/network-indyv3.yaml

```diff
@@ -99,11 +140,11 @@ network:
           node:
             port: 9711
             targetPort: 9711
-            ambassador: 9711
+            ambassador: 9711        # Port for ambassador service
```

Comment:
Shouldn't this ambassador port correspond to the ambassador port that is opened at env.ambassadorPorts

Reply:
In Indy we get additional ambassador ports from this field, so I think we don't need the env.ambassadorPorts at all in this example